Blogpost about documentation move (#125)
wallabag/website: pages/news/20260612-documentation-move/index.md

---

> title: 'We moved our user documentation'

Reviewer: I propose a rewording to avoid confusion: there is actually no change for the users

```suggestion
title: 'Change on the documentation repository'
```

draft: true
date: 2026-06-12 14:00:00 +02:00
description:
---

> Recently, we decided [to archive our documentation repository](https://github.com/wallabag/doc). But don't worry, the wallabag doc is not dead: we moved it [into the main repository of the project](https://github.com/wallabag/wallabag/tree/master/doc).

Reviewer: ```suggestion
Recently, we decided to [merge the documentation repository into the wallabag code repository](https://github.com/wallabag/wallabag/pull/8750). This will greatly simplify contribution workflows by avoiding to maintain and contribute to a separate repository.
```

> With this change, it's easier to have a documentation per version. The contributors who add or change features in the project can now easily update it, without editing an other project.

Reviewer: ```suggestion
This change will also enable us to easily maintain documentations dedicated to minor versions.
```

> One thing that hasn't changed, the place where you can read this documentation: [https://doc.wallabag.org/](https://doc.wallabag.org/)!

Reviewer: ```suggestion
The documentation for the current release is still accessible on [https://doc.wallabag.org/](https://doc.wallabag.org/). Contributions can now be done on [https://github.com/wallabag/wallabag/tree/master/doc](https://github.com/wallabag/wallabag/tree/master/doc).

Finally as a consequence of this merge, we will archive the former doc repository.
```
